ingest/src/main/scala/hydra.ingest/app/AppConfig.scala

```diff
@@ -122,7 +122,7 @@ object AppConfig {
       (
         env("KAFKA_CONSUMER_GROUPS_INTERNAL_TOPIC_NAME").as[String].default("__consumer_offsets"),
           env("HYDRA_CONSUMER_GROUPS_COMMON_CONSUMER_GROUP").as[ConsumerGroup].default("kafkaInternalConsumerGroupsTopic-ConsumerGroupName"),
-            env("CONSUMER_GROUPS_CONSUMER_ENABLED").as[Boolean].default(false)
+            env("CONSUMER_GROUPS_CONSUMER_ENABLED").as[Boolean].default(true)
```

Comment:
Let's make sure to test this in stage with our Load Tester to ensure all is well with the deployment before we push to prod with this change.

Reply:
Definitely.